DigitalLearningSolutions.Data.Migrations/202201120821_ChangeUniqueConstraintsOnCompetencyAssessmentQuestionRoleRequirements.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202201120821)]
+    public class ChangeUniqueConstraintsOnCompetencyAssessmentQuestionRoleRequirements: Migration
+    {
+        public override void Up()
+        {
+            Delete.PrimaryKey("PK_CompetencyAssessmentQuestionRoleRequirements")
+                .FromTable("CompetencyAssessmentQuestionRoleRequirements");
+            Alter.Table("CompetencyAssessmentQuestionRoleRequirements").AddColumn("ID").AsInt32().NotNullable().Identity();
+            Create.PrimaryKey("PK_CompetencyAssessmentQuestionRoleRequirements")
+                .OnTable("CompetencyAssessmentQuestionRoleRequirements").Column("ID");

```

Comment:
You might notice that this is not identical to the Down() method in the other migration. I only noticed when creating this migration that adding a column with `.PrimaryKey()` when altering a table (rather than creating) doesn't actually make the column a primary key. This seems to be due to a subtlety in FluentMigrator that should probably be more obvious but isn't, see here for details: https://github.com/fluentmigrator/fluentmigrator/issues/96